docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -193,12 +193,20 @@ test('Drag test', async t => {
 
 ## Upload Files
 
-Actions that allow you to interact with file upload input elements.
+Actions that allow you to interact with file upload input elements:
```

Comment:
`interact` is bad word in this case. It implies some visual cooperation.
Also, the phrase that I should avoid native browser dialogs confused me.
Remark for '> See the [Test File Upload]' is very important and it may get lost.

I propose such text:
```js
Actions that allow you to manipulate the state of file upload input elements:

[t.setFilesToUpload](../../reference/test-api/testcontroller/setfilestoupload.md)
[t.clearUpload](../../reference/test-api/testcontroller/clearupload.md)

Use the [t.setFilesToUpload](../../reference/test-api/testcontroller/setfilestoupload.md) action to populate the input with the files you want to upload. If your application sends files as soon as you populate the field, the upload occurs when you call the action. Otherwise, your should initiate the upload manually (for example [click](../../reference/test-api/testcontroller/click.md) the button responsible for file submit).

Use the [t.clearUpload](../../reference/test-api/testcontroller/clearupload.md) action to clear the list files to upload.

See the [Test File Upload](../../recipes/basics/test-file-upload.md) recipe for more details.

> The t.setFilesToUpload action only works with `<input type="file">` elements because only these HTML elements are used by browser for choosing files to upload.



```